source/amazon/services/troubleshooting.rst

```diff
@@ -8,46 +8,232 @@ Troubleshooting
+
+**Solution**
+
+No credentials were provided to attempt to access to CloudWatch Logs or that the credentials provided don't grant access to CloudWatch Logs. Check the :ref:`Configuring AWS credentials <amazon_credentials>` section to learn more about the different options available and how to configure them.
+
+
+There are no AWS alerts present in the Wazuh UI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The module is running but no alerts are displayed in the Wazuh UI.
+
+**Solution**
+
+First of all, make sure there is no ``ERROR`` or ``WARNING`` messages in the ``ossec.log`` file by :ref:`enabling debug mode <aws_debug_mode>`. If the module is running as expected but no alerts are being generated it could mean there is no reason for alerts to be raised in first place. Check the following to verify this:
+
+- **Make sure there is data available for the given date**
+
+        When running, the module requests AWS for the logs corresponding to the date indicated using the :ref:`only_logs_aws_buckets` parameter. If this parameter is not specified, it will try to obtain the logs corresponding to the day of execution. Make sure you are specifying a value for :ref:`only_logs_aws_buckets` and that there is data available for that particular date. Check the :ref:`amazon_considerations` page to learn more about how to properly filter the logs using the ``only_logs_after`` parameter.
+
+- **Check if the events are being sent to the analysis engine:**
+
+        A common scenario is that no alerts are being generated because the events are not matching any of the available rules. Take a look to the :ref:`aws_events_processed` section to learn how to check if the AWS logs are being sent to the analysis engine.
+
+
+CloudWatch Logs integration is running but no alert is shown in the Wazuh UI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The module is running without any error or warning messages, but no alerts from CloudWatch Logs are displayed in the Wazuh UI.
+
+**Solution**
+
+A common scenario is that no alerts are being generated because the events are not matching any of the available rules. Take a look to the :ref:`aws_events_processed` section to learn how to check if the AWS logs are being sent to the analysis engine.
+
+Take into account that Wazuh does not provide default rules for the different logs that can be found in CloudWatch Logs, since they can have any type of format and come from any source. Because of this, if a user wants to make use of this integration to be able to process any custom log they will most likely have to configure their own rules for them and even their own decoders, depending on the format of the logs. Take a look at the :ref:`ruleset_custom` section to learn more about this topic.
```

Comment:
Since CloudWatch Logs events come have `json` format users don't need to write any decoder for them.
```suggestion
Take into account that Wazuh does not provide default rules for the different logs that can be found in CloudWatch Logs, since they can have any type of format and come from any source. Because of this, if a user wants to make use of this integration to be able to process any custom log they will most likely have to configure their own rules for them. Take a look at the :ref:`ruleset_custom` section to learn more about this topic.
```

Reply:
Fixed